eventstore/manager.go

```diff
@@ -105,11 +105,18 @@ func (m *Manager) GetStore(id uuid.UUID) *Store {
 }
 
 // Close all the in-mem stores.
-func (m *Manager) Close() (err error) {
+func (m *Manager) Close() error {
```

Comment:
Decided the most reasonable thing is try to continue closing things even of something fails in the middle.
Each failing thing is logged, and the error returned is the last error that happened.
I think this may be better than just returning on first error since would leave other things open that might be closable.